server/routes/topic_page.py

```diff
@@ -75,12 +78,20 @@ def topic_page(topic_id=None, place_dcid=None):
     topic_place_config = topic_configs[0]
     topic_place_config.metadata.place_dcid.append(place_dcid)
     # Populuate comparison places (fixed places) for all tiles
+    comparison_places = [place_dcid]
+    more_places = request.args.getlist('places')
+    if more_places:
+      comparison_places.extend(more_places)
+    for p in _SDG_COMPARISON_PLACES:
+      if p not in comparison_places:
+        comparison_places.append(p)
+        if len(comparison_places) == _MAX_NUM_PLACES:
```

Comment:
nit: check this first in each step of the for-loop (before `if p not in comparison_places:`) because comparison_places could already have a length of 6 when we enter the for loop?

Reply:
good point